src/ConfirmPrompt.php

```diff
@@ -49,4 +51,17 @@ public function label(): string
     {
         return $this->confirmed ? $this->yes : $this->no;
     }
+
+    /**
+     * Configure the default fallback behavior.
+     */
+    protected function configureDefaultFallback(): void
+    {
+        self::fallbackUsing(fn (self $prompt) => $this->retryUntilValid(
+            fn () => (new SymfonyStyle(new ArrayInput([]), static::output()))->confirm($prompt->label, $prompt->default),
```

Comment:
We'd need to call the `setInteractive` method on the `Input` object to respect the `--no-interaction` option.

Implementing a first-party "non-interactive" feature in Prompts is probably worthwhile so we can return an appropriate value where possible and ensure that any `required` and `validate` arguments would pass regardless of the fallback configuration.

Symfony's choice component, for example, does not allow you to select no option but will return `null` in non-interactive mode.